src/gluonts/evaluation/_base.py

```diff
@@ -332,7 +332,7 @@ def get_metrics_per_ts(
 
         median_fcst = forecast.quantile(0.5)
         seasonal_error = calculate_seasonal_error(
-            past_data, forecast, self.seasonality
+            past_data, forecast.start_date.freqstr, self.seasonality
```

Comment:
How about we create a property on `Forecast` that returns `start_data.freq`?

Reply:
ah yes... sometimes i get confused when to use `freq` and when to use `freqstr`... but good idea let me do that